sdks/python/apache_beam/io/sdf_restriction_provider.py

```diff
@@ -0,0 +1,56 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from __future__ import absolute_import
+
+from apache_beam.io.restriction_trackers import SDFBoundedSourceRestrictionTracker
+from apache_beam.transforms.core import RestrictionProvider
+
+__all__ = ['SDFBoundedSourceRestrictionProvider']
+
+
+class SDFBoundedSourceRestrictionProvider(RestrictionProvider):
```

Comment:
Ditto on making this a class defined within SDFBoundedSourceWrapper and not expose it to users.